src/lib/components/workflow/filter-search/list-filter.svelte

```diff
@@ -22,10 +22,12 @@
 
 <div class="flex">
   <ChipInput
+    label={translate('type-or-paste-in')}
```

Comment:
Would it be helpful to include the attribute here as well?

```suggestion
    label="{translate('type-or-paste-in')} {$filter.attribute}"
```

or even just the attribute

```suggestion
    label="{$filter.attribute}"
```

Reply:
I like just using the attribute, good call 👍